### PR TITLE
Updating dash

### PR DESCRIPTION
**Pull request by catherine-martlin**
Updates to DASH to allow running the notebook in the new `stenv` environment.

### CODE REVIEW

**Comment by catherine-martlin** [comment]
Oops, forgot to make this on my Fork - let me know if that's preferred or if I should just do updates from my fork in the future!

**Comment by catherine-martlin** [comment]
Okay! The DASH notebook now works when used with this the `stenv-macOS-py3.10-2022.12.01-stable.yml` although it does require one extra `conda install` of `Cython` in order for `lacosmicx` to build/run. We can either add that to the general installation or we can have a note in the DASH README to `conda install Cython` before the `lacosmicx` installation.

**Comment by FDauphin** [comment]
> Oops, forgot to make this on my Fork - let me know if that's preferred or if I should just do updates from my fork in the future!

That's fine, I think moving from (fork -> main) to (branch -> main) will make my work flow a lot easier.

**Comment by FDauphin** [comment]
> Okay! The DASH notebook now works when used with this the `stenv-macOS-py3.10-2022.12.01-stable.yml` although it does require one extra `conda install` of `Cython` in order for `lacosmicx` to build/run. We can either add that to the general installation or we can have a note in the DASH README to `conda install Cython` before the `lacosmicx` installation.

It would be best to add the `Cython` installation to the DASH README.md

File: notebooks/dash/dash.ipynb [hunk 1]
```diff
@@ -754,18 +754,18 @@
...
+    "**Updated on:** 2023-01-25\n",
     "\n",
     "<a id=\"cite\"></a>\n",
     "## Citations\n",
     "\n",
-    "If you use `numpy`, `matplotlib`, `astropy`, `astroquery`, or `drizzlepac` for published research, please cite the\n",
-    "authors. Follow these links for more information about citing the libraries below:\n",
+    "If you use `numpy`, `matplotlib`, `astropy`, `astroquery`, `photutils`, or `drizzlepac` for published research, please cite the authors. Follow these links for more information about citing the libraries below:\n",
     "\n",
     "* [Citing `numpy`](https://www.scipy.org/citing.html#numpy)\n",
```
Comment (FDauphin) [inline notebooks/dash/dash.ipynb:764]: Numpy has a new citation link: https://numpy.org/citing-numpy/

**Comment by catherine-martlin** [comment]
@FDauphin - I added the instructions to install `cython` to the README for the DASH notebook

**Review by FDauphin — COMMENTED** [review]
General comment: Why did you change the format of the section titles? It breaks consistency across the other notebooks. Besides that all the code works.

File: notebooks/dash/dash.ipynb (hunk 1 above)
Comment (catherine-martlin) [reply notebooks/dash/dash.ipynb:764]: Ohhh, I'm so glad you said that about the format of the section titles - I think it's something in my jupyter notebook settings and it happened automatically and I think it's not pretty at all but thought it was purposeful as I hadn't changed anything manually and was going to leave it be. 

I'll go check and see if I can get it to reset to not have all those numbers and to match the other notebooks formatting.

**Comment by catherine-martlin** [comment]
@FDauphin - can you check the section title formatting now? I added a notebook extension that I needed to have turned off and I think I just sent up the correct formatting.

**Comment by catherine-martlin** [comment]
Oh, and we need an updated copy of the dash.html file from the new notebook!

**Comment by FDauphin** [comment]
@catherine-martlin everything looks good. Final edits:
- I also noticed matplotlib citation is old so use [this link instead](https://matplotlib.org/stable/users/project/citing.html).
- new html file
Once these are complete we can merge, can you finish this my COB tomorrow?

**Comment by catherine-martlin** [comment]
@FDauphin - I think I got it all!

**Review by FDauphin — APPROVED** [review]
Everything is good. Approving for merge

**Pull request merged**